common/scala/src/main/scala/whisk/core/entity/EntityPath.scala

```diff
@@ -45,9 +45,30 @@ protected[core] class EntityPath private (val path: Seq[String]) extends AnyVal
     def addpath(e: EntityName) = EntityPath(path :+ e.name)
     def root = EntityPath(Seq(path(0)))
     def last = EntityName(path.last)
+    def dropLast = EntityPath(path.dropRight(1)) // remove last element from path (e.g., name)
+    def defaultPackage = path.size == 1   // if only one element in the path, then it's the namespace with a default package
```

Comment:
@markusthoemmes I completely agree we abuse this type, and in fact, at some point we do need to do a pass and remove any abusive statement since we introduced FQENs. But I'm stuck, I need to figure out whether something needs to resolve a binding or not (i.e., if it's in the default package or not). One option could be to push this code to the only place where it's used to not proliferate this check. But other than that, I don't have a better idea. Suggestion?